### DISCUSSION TITLE
Welcome to I4G-OPENSOURCE-FRONTEND-PROJECT-2022 Discussions!

### DISCUSSION THREAD

**Discussion by ingressive4good** [Ingressive-for-Good/I4G-OPENSOURCE-FRONTEND-PROJECT-2022#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by codewithmide** [comment]
Hello Open Sorcerers🧙‍♀️,

The I4G community appreciates your amazing contributions and would like to say a very big thank you for your hard work, you all are doing very great.😁

However, regarding recent events, we have discovered that some of your PR's are having merge conflicts. We know how frustrating this can be. So we  have decided to address this issue and let you know how you can fix them and some other related problems.👇

- Always ensure you do a git pull on the `develop` branch before making your changes.🙏
- Please make sure that your branch name is be related to the problem you are solving.♻️
- If you're optimising a section of code or you're fixing a bug. Please always attach a screenshot of your changes and that of the existing one. This will ease the work of the maintainers when deciding if your changes can be merged or not.📸
- If you will be working on an issue raised by someone else, please ensure you speak to them or any of the maintainers about it first.🤷‍♂️

There are still more than 20 active issues, kindly signify if you want any of them to be assigned to you.🙋‍♂️

Once more, we appreciate your resilience and the effort you're putting in place to make this project a successful one. Thank you so much LFG!!!🚀

**Comment by JohnChizzy** [comment]
please I would like to be assigned to one of those issues. I have not gotten any yet

**Reply by codewithmide** [reply]
> please I would like to be assigned to one of those issues. I have not gotten any yet

@JohnChizzy Thank you for your interest! Please signify under any of the issues you have an interest in. It will be assigned to you.
